tests/runtime/variable

```diff
@@ -40,7 +40,7 @@ EXPECT openflags: 64
 AFTER ./testprogs/syscall openat
 
 NAME tracepoint arg casts in predicates
-RUN {{BPFTRACE}} -e 'tracepoint:syscalls:sys_enter_wait4 /args.ru/ { @ru[tid] = args.ru; } tracepoint:syscalls:sys_exit_wait4 /@ru[tid]/ { @++; exit(); }' -c ./testprogs/wait4_ru
+RUN {{BPFTRACE}} -e 'tracepoint:syscalls:sys_enter_wait4 /args.ru/ { @ru[tid] = args.ru; } tracepoint:syscalls:sys_exit_wait4 /has_key(@ru, tid)/ { @++; exit(); }' -c ./testprogs/wait4_ru
```

Comment:
```suggestion
RUN {{BPFTRACE}} -e 'tracepoint:syscalls:sys_enter_wait4 /args.ru/ { @ru[tid] = args.ru; } tracepoint:syscalls:sys_exit_wait4 /@ru[tid]/ { @++; exit(); }' -c ./testprogs/wait4_ru
```

This change looks wrong to me. The test is specifically checking that pointer values (such as `@ru[tid]`) can be used in predicates

Also, we don't need to swap out instances of this pattern everywhere - it's not going to be deprecated and isn't incorrect.

Reply:
oops, got a little replacement happy here. Will change it back.